actors/builtin/miner/miner_test.go

```diff
@@ -1703,6 +1706,67 @@ func TestTerminateSectors(t *testing.T) {
 			assert.Equal(t, big.Zero(), st.InitialPledgeRequirement)
 		}
 	})
+
+	t.Run("charges correct fee for young termination of committed capacity upgrade", func(t *testing.T) {
+		actor := newHarness(t, periodOffset)
+		rt := builderForHarness(actor).
+			WithBalance(bigBalance, big.Zero()).
+			Build(t)
+		actor.constructAndVerify(rt)
+
+		// A miner will pay the minimum of termination fee and locked funds. Add some locked funds to ensure
+		// correct fee calculation is used.
+		actor.addLockedFunds(rt, big.Mul(big.NewInt(1e18), big.NewInt(20000)))
+
+		// Move the current epoch forward so that the first deadline is a stable candidate for both sectors
+		rt.SetEpoch(periodOffset + miner.WPoStChallengeWindow)
+
+		// Commit a sector to upgrade
+		// Use the max sector number to make sure everything works.
```

Comment:
Why use the max sector number?  Is this testing the abi constant or avoiding possible conflict with previously set up testing state?

Reply:
This comment was stolen from another test. I'll set sector number to 1.